util/regexp-assemble/data/933161.data

```diff
@@ -24,7 +24,7 @@
 
 ##!+i
 ##!^\b
-##!$(?:\s|/\*.*\*/|//.*|#.*)*\(.*\)
+##!$\(.*\)
```

Comment:
After removing whitespace and comment markers you might miss this:
```
=abs\n#dummy comment\n('gotcha')
```
which would be converted to:
```
=absdummycomment('gotcha')
```
Removing whitespace could be ok I guess but removing comment markers will break the rule.